diff --git a/scripts/build/termux_download_ubuntu_packages.sh b/scripts/build/termux_download_ubuntu_packages.sh
index e89b630..e5ee96e 100755
--- a/scripts/build/termux_download_ubuntu_packages.sh
+++ b/scripts/build/termux_download_ubuntu_packages.sh
@@ -11,7 +11,7 @@ termux_download_ubuntu_packages() {
 			echo "DESTINATION    [\$TERMUX_PKG_HOSTBUILD_DIR/ubuntu_packages]"
 			echo "ARCHITECTURE   [amd64]"
 			echo "MIRROR_URL     [https://archive.ubuntu.com/ubuntu]"
-			echo "UBUNTU_RELEASE [noble] # latest Ubuntu LTS"
+			echo "UBUNTU_RELEASE [resolute] # latest Ubuntu LTS"
 			return 1
 		}
 
@@ -26,7 +26,7 @@ termux_download_ubuntu_packages() {
 	: "${DESTINATION:="$TERMUX_PKG_HOSTBUILD_DIR/ubuntu_packages"}"
 	: "${ARCHITECTURE:=amd64}"
 	: "${MIRROR_URL:="https://archive.ubuntu.com/ubuntu"}"
-	: "${UBUNTU_RELEASE:=noble}" # Default to latest Ubuntu LTS, currently 24.04 Noble Numbat
+	: "${UBUNTU_RELEASE:=resolute}" # Default to latest Ubuntu LTS, currently 26.04 Resolute Raccoon
 
 	mkdir -p "$DESTINATION"
 
